src/profiles/geojson.json

```diff
@@ -0,0 +1,208 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "id": "https://raw.githubusercontent.com/fge/sample-json-schemas/master/geojson/geojson.json#",
```

Comment:
Not sure if you investigated them thoroughly but this one seems to be a bit more compliant:

https://github.com/SchemaStore/schemastore/blob/master/src/schemas/json/geojson.json

eg it allows position to have more than 2 values (GeoJSON allows a third one for elevation). Both allow the `crs` object which has been removed in the latest GeoJSON spec version, but that should be all right.